src/components/Dropdown/Dropdown.tsx

```diff
@@ -242,7 +242,7 @@ function useDropdownContextInstance(
   value: string,
   index: number,
   focusedItemData: Record<string, unknown> | undefined,
-  screenReaderUUID: string,
+  screenReaderUUID: string | undefined,
```

Comment:
why is undefined allowed here?

Reply:
because `useId()` can return undefined on first render

Reply:
Oh I see it's for the server